working-together/README.md

```diff
@@ -93,6 +97,12 @@ You can [download a double-sided printout] of these guides to display in your me
     - Always try to be explicit rather than implicit with regards to expressing emotions or intent,
       especially in text-based communication (where emojis and tone tags can be helpful).
       When others' intent is not explicit, assume positive or neutral intent instead of negative intent.
+- **Be mindful of power dynamics:** Be aware of power dynamics that may be
+  present in the meeting, such as those based on race, gender, or seniority.
+  Ensure that all participants have an opportunity to share their perspectives,
+  respectfully disagree, and that their contributions are valued equally.
+  - If you are in a position of power, consider speaking last to ensure that
```

Comment:
This is a great addition. Thanks for adding it.

Reply:
This looks good to me! Great job @clarissalimab!